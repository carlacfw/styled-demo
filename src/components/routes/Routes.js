import React from 'react'
import { Route } from 'react-router'
import styled from 'styled-components'
import Form from '../form/Form'
import Props from '../props/Props'
import Home from '../home/Home'

const RoteWrapper = styled.div`
  display: flex;
  flex-direction: column;
  flex: 1;
  align-items: center;
`

const Routes = () => {
  return (
    <RoteWrapper>
      <Route exact path="/" component={Home} />
      <Route path="/form" component={Form} />
      <Route path="/props" component={Props} />
    </RoteWrapper>
  )
}

export default Routes
