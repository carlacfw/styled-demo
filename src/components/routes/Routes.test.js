import React, { Component } from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Routes from './Routes'
describe('The Home page ', () => {
  it('It renders the routesu correctly', () => {
    const wrapper = shallow(<Routes />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
