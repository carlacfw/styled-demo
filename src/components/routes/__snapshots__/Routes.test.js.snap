// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Home page  It renders the routesu correctly 1`] = `
<styled.div>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    path="/form"
  />
  <Route
    component={[Function]}
    path="/props"
  />
</styled.div>
`;
